frontend/components/Nav/Nav.tsx

```diff
@@ -0,0 +1,11 @@
+const Nav: React.FC = () => {
+    return (
+        <nav role="navigation">
+            <div aria-label="site logo">
+                <h2>Tutorials</h2>
+                <h4>by DevIsle</h4>
```

Comment:
Why not `h1` and `h2`?

Reply:
h1 looks big. Still working on it.

Reply:
Yeah, just for accessibility it's not the best choice. Maybe create a class?

Reply:
Endormi you're completely right, see this: https://www.w3.org/WAI/tutorials/page-structure/headings/
The accessibility crawler views them in levels of 'importance'. Hope this helps in addition to @endormi 's comment.

Reply:
Using `styled-components` removes the need to use classes. Accessibility comes to play in that matter? didn't know.

Reply:
Ok, I haven't really used `styled-components`.

Reply:
@endormi yeah looks like it kind of does and more than that it's a great practise. Thanks mate.
@ale8k you too alexandar the great.